src/api/rest/providers/github/schemas/repos.js

```diff
@@ -0,0 +1,162 @@
+import { schema } from 'normalizr';
+import { initSchema } from 'utils';
+
+import { userSchema } from './users';
+import { orgSchema } from './orgs';
+
+const isInMinimalisticForm = entity => typeof entity.full_name === 'undefined';
+
+export const repoSchema = new schema.Entity(
+  'repos',
+  {
+    userOwner: userSchema,
+    orgOwner: orgSchema,
+  },
+  {
+    idAttribute: repo =>
+      (isInMinimalisticForm(repo) ? repo.name : repo.full_name).toLowerCase(),
+    processStrategy: entity => {
+      const processed = initSchema();
+
+      // Repo received from events
+      if (isInMinimalisticForm(entity)) {
+        processed.id = entity.name.toLowerCase();
+        processed.fullName = entity.name;
+        processed.shortName = entity.name.substring(
```

Comment:
It is possible have more than one '/' in the string ?

Reply:
not for a repo